### ISSUE TITLE
The split of training and validation.

### ISSUE THREAD

**Issue by scuwyh2000** [yunxiangfu2001/DreamDA#1]
Hi, great work!

I wanna ask the ratio of training over validation in the expand datasets.

**Comment by yunxiangfu2001** [comment]
Hi, I don't understand your question.  

Nevertheless, we generate synthetic data with a scale of x10, x20, and x30 compared to the size of the original real training data. The expanded dataset is a combined set of synthetic and real images. Specific settings are presented in our paper.

Hope this helps!

**Comment by Yanghao-Wang** [comment]
> Hi, I don't understand your question.
> 
> Nevertheless, we generate synthetic data with a scale of x10, x20, and x30 compared to the size of the original real training data. The expanded dataset is a combined set of synthetic and real images. Specific settings are presented in our paper.
> 
> Hope this helps!

Hi, I mean the split of the datasets. When you are training the classifier (which is resnet50 in your paper), you do not use a validation set to monitor the training process? If you do, what is this validation set.

**Comment by yunxiangfu2001** [comment]
Hi, we use the official train-validation-test splits provided when downloading the datasets, which consist of real images. If no validation set is given, we do not use a validation set during training. For the ShenzhenTB dataset, no splits were provided so we randomly partitioned the dataset into train, validation, and test sets with a ratio of 7:1:2.
